public/landingPage.html

```diff
@@ -5,95 +5,40 @@
-                max-width: 400px;
-            }
-
-            #main-view-container {
-                height: 100%;
-                width: 100%;
-                padding-left: 100px;
-                text-align: left;
-            }
-
-            h3 {
-                color: #ffffff;
-            }
-
-            p {
-                margin-bottom: 30px;
-                color: #efefef;
-            }
-
-            #tag {
-                text-align: left;
-            }
-
-            #background-overlay {
-                position: absolute;
-                background: #555;
-                width: 100%;
-                height: 100%;
-                z-index: 900;
-                opacity: 0.7;
-            }
-
-            #title {
-                margin-bottom: 30px;
-            }
-
-            .hidden {
-                display: none;
-            }
-
-            .sunset-background {
-                background-image: url(/assets/sunrise.jpg);
-            }
-        </style>
-
-        <script>
-        window.onload = function() {
-            if (document.cookie != "") {
-                window.location.href = "/login";
-
-            } else {
-                document.body.className += "sunset-background";
-                document.documentElement.className += "sunset-background";
-                hiddenElements = document.getElementsByClassName("hidden");
-                while (hiddenElements.length > 0) {
-                    var element = hiddenElements[0];
-                    element.className = element.className.replace(/hidden/, "");
-                    hiddenElements = document.getElementsByClassName("hidden");
-                }
-            }
-        }
-        </script>
-
+        <link rel="stylesheet" type="text/css" href="/assets/landingPage.css"/>
     </head>
     <body>
         <div id="background-overlay" class="hidden"></div>
         <div id="main-view-container" class="container hidden">
             <div id="main-view" ui-view="content">
-                <h3 id="title">PenPal Gladiators</h3>
+                <h3 id="title">PenPal Gladiator Project</h3>
                 <p id="tag">
-                    Politics is a complicated topic. People have disagreements and it's hard to agree on something. But what if you could talk about it? To be able to discuss differences with someone of opposing opinion?
+                    Engage in constructive and open discourse about political
+                    topics that are important to you.
+                    <!--
+                    Politics is a complicated topic. People have disagreements
+                    and it's hard to agree on something. But what if you could
+                    talk about it? To be able to discuss differences with
+                    someone of opposing opinion?
+                    -->
                 </p>
                 <p>
+                    In exchange for deliberative listening, see the world from
```

Comment:
**Warning to whoever merges this in:**
If you're seeing this message in a non-outdated diff, this means that the commit corresponding to `Landing page overhaul` is still (erroneously) in the history for this branch.  This commit has already been merged in to `iteration4` so you should NOT see this when you're about to merge/etc.  Ping me if this is confusing.